Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.yml

```diff
@@ -3592,6 +3592,24 @@ script:
       Update one or more alerts with the provided arguments.
       Required license: Cortex XDR Prevent, Cortex XDR Pro per Endpoint, or Cortex XDR Pro per GB.
     name: xdr-update-alert
+  - arguments:
```

Comment:
Since it uses the same functionality as the xdr-get-distribution-url command, our TPM has approved the following approach: delete the new xdr-download-distribution command, and instead add an argument called download_package (defaulting to false) to xdr-get-distribution-url. When download_package is set to false, the command will return only the URL; when set to true, it will perform the GET request that was added in your new xdr-download-distribution command.
Please apply this change in cortexcoreir as well.